hibernate-jcache/src/test/java/org/hibernate/orm/test/jcache/RefreshUpdatedDataTest.java

```diff
@@ -8,12 +8,12 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import javax.persistence.ElementCollection;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.Id;
-import javax.persistence.Table;
-import javax.persistence.Version;
+import jakarta.persistence.ElementCollection;
```

Comment:
I've been a bit inconsistent with import ordering... Whats the protocol when choosing between minimized changes and correct import order?

Reply:
We don't really have a hard/fast wrt "minimizing clonflicts".  No one is likely to have touched the files under hibernate-jcache though, so this is fine.

Ordering is defined by our importable code styles for IJ and Eclipse IDEs